scripts/upload_ci_stats.sh

```diff
@@ -17,8 +10,10 @@ fi
 
 PSQL=(psql "${CI_STATS_DB}" -qtAX "--set=ON_ERROR_STOP=1")
```

Comment:
This looks like a typo, but not sure.

Reply:
Which part exactly? Looks correct to me...

Reply:
I see below that it is creating a list and then expanding it, so it seems correct, although it is one of the "interesting" features I mention that is probably hard to maintain. First, it seemed to be missing a `$`, but that would not work either.

Reply:
Well, that's the usual bash thing, using arrays to save a command line. It's normally done this way not to mangle the arguments that have spaces in them. For this case it doesn't matter I guess. The `$` -- you mean before `ON_ERROR_STOP` or where? `ON_ERROR_STOP` is a psql internal variable, I'm setting it to true so that it fails on errors inside an SQL script.

People are always telling me that about bash :) I guess I'll try to write the next script in Python. It's inconvenient when you need to just run some programs and capture output, but it has much less weird stuff, I agree.

Reply:
> Well, that's the usual bash thing, using arrays to save a command line. It's normally done this way not to mangle the arguments that have spaces in them. For this case it doesn't matter I guess. The `$` -- you mean before `ON_ERROR_STOP` or where? `ON_ERROR_STOP` is a psql internal variable, I'm setting it to true so that it fails on errors inside an SQL script.

Yeah, that is what I realized.

> People are always telling me that about bash :) I guess I'll try to write the next script in Python. It's inconvenient when you need to just run some programs and capture output, but it has much less weird stuff, I agree.

Not important at this point, but yes, bash has a lot of weird stuff that can be hard to follow. :)